internal/custom_executable/yes/yes_test.go

```
@@ -0,0 +1,133 @@
+		for scanner.Scan() && count < 40_000 {
+			lines = append(lines, scanner.Text())
+			count++
+		}
+	}()
+
+	// Wait for either context timeout or reading to complete
+	select {
+	case <-ctx.Done():
+		// Context timed out, ensure the process is terminated
+		if cmd.Process != nil {
+			cmd.Process.Kill()
```

golangci-lint (1.64.8)

Error return value of `cmd.Process.Kill` is not checked

(errcheck)